File: codeforces/low_difficulty/difficulty_800/22.anton_and_danik_[734A]/aad.cpp
#include <iostream>
#include <cstring>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned int n, games_won_anton = 0, games_won_danik = 0;
  string s;
  cin >> n;
  cin >> s;
  while (n--) {
    if (s[n] == 'A') {
      games_won_anton++;
    }
    else {
      games_won_danik++;
    }
  }
  if (games_won_anton > games_won_danik) {
    cout << "Anton" << endl;
  }
  else if (games_won_anton < games_won_danik) {
    cout << "Danik" << endl;
  }
  else {
    cout << "Friendship" << endl;
  }
  return 0;
}

File: codeforces/low_difficulty/difficulty_800/22.anton_and_danik_[734A]/aad.py
# Author: José Rodolfo (jric2002)
n = int(input())
s = str(input())
games_won_anton = 0
games_won_danik = 0
for position in range(0, n):
  if (s[position] == 'A'):
    games_won_anton += 1
  else:
    games_won_danik += 1
if (games_won_anton > games_won_danik):
  print("Anton")
elif (games_won_anton < games_won_danik):
  print("Danik")
else:
  print("Friendship")